os/linux_base.h

```diff
@@ -38,6 +38,8 @@ typedef uint32_t mach_msg_return_t;
 
 typedef uintptr_t mach_vm_address_t;
 
+typedef uint32_t mach_msg_bits_t;
```

Comment:
all this should move to linux_stubs.h because linux_base.h is a public API header that should not define/export these types that are only used to build dispatch.